### ISSUE TITLE
Heat map squeezed 

### ISSUE THREAD

**Issue by PrimozGodec** [paulo-raca/highcharts-contour#8]
Sometimes (when data range is longer along x axis than y) happens that heat map is squeezed along y axis. After quick look in code I noticed that there should be problem with positions of point in graph. It seems that wrong positions appears by heatmap by Highchart. Example:
https://jsfiddle.net/ra2fum9y/

**Comment by PrimozGodec** [comment]
It is connected with that issue for heatmap https://github.com/highcharts/highcharts/issues/5492. 
I solved the problem with following settings, but it seems that they gonna solve the problem with automatic setting of parameters `colsize` and `rowsize`. 

```
"plotOptions": {
        "contour": {
            "colsize": 0.1, 
            "rowsize": 0.1
        }
    },
```

**Comment by paulo-raca** [comment]
Thanks for reporting it, I've fixed the issue

**Comment by historicbruno** [comment]
This fix seems to have broken tooltip colors: #10
